contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile:cairo": "hardhat starknet-compile",
+    "compile:cairo": "scarb --profile release build",
     "compile:solidity": "hardhat compile",
     "compile": "yarn compile:cairo && yarn compile:solidity",
```

Comment:
So this should then be removed too, instead `scarb ... && yarn compile:solidity` can be called in the makefile